package/src/types/v1-store-offers.ts

```diff
@@ -0,0 +1,27 @@
+export interface Cost {
+	"85ad13f7-3d1b-5128-9eb2-7cd8ee0b5741": number
```

Comment:
This is bad typing

Reply:
See my comment

Reply:
Yeah I think this is ok, it wouldn't change.